tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "noUnusedParameters": false,
     "noUnusedLocals": false,
     "noImplicitReturns": true,
-    "emitDecoratorMetadata": true,
```

Comment:
this line got removed and might be causing the Jest unit tests to fail on CircleCI

![image](https://user-images.githubusercontent.com/643976/119674499-3d294f80-be0a-11eb-9829-3e6b0eb34218.png)